src/cli/cli_config.rs

```diff
@@ -18,6 +22,60 @@ pub struct ProjectConfig {
     pub manifest_path: Option<PathBuf>,
 }
 
+/// Channel configuration
+#[derive(Parser, Debug, Default)]
+pub struct ChannelsConfig {
+    /// The channels to consider as a name or a url.
+    /// Multiple channels can be specified by using this field multiple times.
+    ///
+    /// When specifying a channel, it is common that the selected channel also
+    /// depends on the `conda-forge` channel.
+    ///
+    /// By default, if no channel is provided, `conda-forge` is used.
+    #[clap(long = "channel", short = 'c')]
```

Comment:
```suggestion
    #[clap(long = "channel", short = 'c', value_name = "CHANNEL")]
```
Nitpick: Otherwise people might think it can except more than one channel.